setpass.py

```diff
@@ -0,0 +1,45 @@
+#   Copyright 2016 Massachusetts Open Cloud
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+import string
+import random
+
+
+def random_password(size):
+    """Generate a random password of length 'size'"""
+    chars = string.ascii_letters + string.digits + string.punctuation[2:6]
```

Comment:
Thanks for the reminder!   Update does the following:
* makes which punctuation marks we're using explicit
* expands the docstring
* added ! as a possible character

In the future I may make it fancy so that you can pass an optional keyword to overwrite what punctuation to use, including None.  But this should do for now.